en/components/grids_templates/validation.md

```diff
@@ -561,16 +541,39 @@ public stateMessage(cell: IgxGridCell) {
     return messages;
 }
 ```
+}
+### Cross-field example
+
+@@if (igxName === 'IgxGrid') {
+
+The below sample demonstrates the cross-field validation in action.
+
+<code-view style="height:560px" 
```

Comment:
Please set more height in all samples so that the commit button at the bottom is visible